index.js

```diff
@@ -0,0 +1,96 @@
+const data = [
+  {
+    id: 1,
+    name: 'Jock Toppas',
+    image: 'img/person1.png',
+    job_title: 'Jock Professor of Entrepreneurial Legal Studies at Harvard Law School',
+    biography: 'Toppas studies commons-based peer production, and published his seminal book The Wealth of Networks in 2006.',
+  },
+  {
+    id: 2,
+    name: 'Jone Dannet',
+    image: 'img/person2.png',
+    job_title: '',
+    biography: 'As the main venue for new media art production in California, Nabi promotes cross-disciplinary collaboration and understanding among science technology, humanities, and the arts.',
+  },
+  {
+    id: 3,
+    name: 'Latour Brigg',
+    image: 'img/person3.png',
+    job_title: 'Assistant Manager',
+    biography: 'Brigg Latour worked in several jobs that included a try as a machinist,[10] several other jobs,[11] and then back to work as a machinist.',
+  },
+  {
+    id: 4,
+    name: 'Nolan Crosoer',
+    image: 'img/person4.png',
+    job_title: 'Executive Design',
+    biography: 'Nolan Crosoer worked in several jobs that included a try as a machinist,[10] several other jobs,[11] and then back to work as a machinist.',
+  },
+  {
+    id: 5,
+    name: 'Amil Fielding',
+    image: 'img/person5.png',
+    job_title: 'Social Worker',
+    biography: 'Amil Fielding worked in several jobs that included a try as a machinist,[10] several other jobs,[11] and then back to work as a machinist.',
+  },
+  {
+    id: 6,
+    name: 'Lark Tom',
+    image: 'img/person6.png',
+    job_title: 'Web Developper ',
+    biography: 'Lark Tom worked in several jobs that included a try as a machinist,[10] several other jobs,[11] and then back to work as a machinist.',
+  },
+];
+
+const speaker = (data) => `<div class="speaker anim">
```

Comment:
- [ ] Let's replace the `div` here with the semantic `article` element for the reasons discussed earlier. 🧐️